### ISSUE TITLE
actual dimensions of humans

### ISSUE THREAD

**Issue by peethamber** [facebookresearch/DensePose#91]
Hi

Is it possible to derive actual dimensions of the human body from any of the densepose inferences. Any pointer in this direction will help.

Thanks

**Comment by nneverova** [comment]
@peethamber, no, DensePose output wouldn't be helpful here.

**Comment by peethamber** [comment]
Thanks for letting me know.
This work is definitely a fantastic step forward and quite helpful for all of us in our research.

Coming back to my earlier question, Is there any remote inference from densepose that I can use as a base to arrive at any meaningful interpretation of real life dimensions.

**Comment by nneverova** [comment]
@peethamber, to get the absolute sizes you would need to know a) parameters of your camera and b) distances between your camera and every person. 

a) is usually not available on standard datasets, but could be easy to obtain if you collect your own data;
b) if the distances are not known in your setup, it can be solved, for example, by introducing an additional depth sensor (or an additional camera looking from a different angle) and either collecting data for training a network or using it directly during inference.

**Comment by peethamber** [comment]
Thanks for the prompt response. To summarize and If I have understood you right

Step 1 : To collect own data and create the dataset with distances from camera.
Step 2 : Train custom built network on data and infer dimensions from output from network.

If so, I would go ahead and do that and close this issue.

**Comment by nneverova** [comment]
Yes, it should work if the camera parameters are also known at test time.

**Comment by zziz** [comment]
@peethamber did you make any progress? How was the result?
